# encoderStream.msg
uint64 enc_timestamp
int32 enc1
int32 enc2
int32 enc3

